ui/core/src/ui-core/autosuggest/AutoSuggest.tsx

```diff
@@ -57,6 +57,19 @@ export interface AutoSuggestProps extends React.InputHTMLAttributes<HTMLInputEle
   /** Gets a label associated with a given value */
   getLabel?: (value: string | undefined) => string;
 
+  /** Use it only if you need to customize the rendering of the input.
+   * @alpha
+   */
+  renderInputComponent?: ReactAutosuggest.RenderInputComponent<AutoSuggestData>;

```

Comment:
renderInputComponent and renderSuggestionsContainer are very specific to the react-autosuggest component. Other than introducing a type that equates to the ReactAutosuggest type, I'm not sure what else we can do. Instead of @alpha, we can change it to @internal and never expose it in our public API.

Reply:
Changed renderInputComponent and renderSuggestionsContainer to @internal